codespell_lib/data/dictionary_rare.txt

```diff
@@ -91,6 +91,7 @@ leaded->led, lead,
 leas->least, lease,
 lightening->lightning, lighting,
 loafing->loading
+lod->load
```

Comment:
Actually a bit late (having just merged), but suggesting loud would also be good, as well as stopping any auto-fixing...